x-pack/plugins/ml/public/ui_actions/apply_influencer_filters_action.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ActionContextMapping, createAction } from '../../../../../src/plugins/ui_actions/public';
+import {
+  AnomalySwimlaneEmbeddable,
+  SwimLaneDrilldownContext,
+} from '../embeddables/anomaly_swimlane/anomaly_swimlane_embeddable';
+import { MlCoreSetup } from '../plugin';
+import { SWIMLANE_TYPE, VIEW_BY_JOB_LABEL } from '../application/explorer/explorer_constants';
+import { Filter, FilterStateStore } from '../../../../../src/plugins/data/common';
+
+export const APPLY_INFLUENCER_FILTERS_ACTION = 'applyInfluencerFiltersAction';
+
+export const CONTROLLED_BY_SWIM_LANE_FILTER = 'anomaly-swim-lane';
+
+export function createApplyInfluencerFiltersAction(
+  getStartServices: MlCoreSetup['getStartServices']
+) {
+  return createAction<typeof APPLY_INFLUENCER_FILTERS_ACTION>({
+    id: 'apply-to-current-view',
+    type: APPLY_INFLUENCER_FILTERS_ACTION,
+    getIconType(context: ActionContextMapping[typeof APPLY_INFLUENCER_FILTERS_ACTION]): string {
+      return 'filter';
+    },
+    getDisplayName() {
+      return i18n.translate('xpack.ml.actions.applyInfluencersFiltersTitle', {
+        defaultMessage: 'Apply influencer filters',
```

Comment:
Possible edit for this text to `Filter for value` would make it consistent with the doc table action for example. Or `Filter for values` for multiple selection (if you keep multiple influencer selection filtering!).

![image](https://user-images.githubusercontent.com/7405507/87298147-9bef1e80-c501-11ea-893a-4d64934a5589.png)

Reply:
Changed the label in https://github.com/elastic/kibana/pull/71082/commits/9693776fae7909a55a96a6852acadd9d3684ce4b